src/main/java/knight2103/tasks/TaskList.java

```diff
@@ -0,0 +1,133 @@
+package knight2103.tasks;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    /**
+     * Constructs an object containing an empty list of tasks stored as an ArrayList.
+     * This list is called the taskList.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Constructs an object containing an existent list of tasks stored as an ArrayList.
+     * This list is called the taskList.
+     */
+    public TaskList(ArrayList<Task> storageData) {
+        this.tasks = storageData;
+    }
+
+    /**
+     * Returns the number of tasks in the bot's taskList.
+     *
+     * @return Number of tasks in the bot's taskList.
+     */
+    public int getSize() {
+        return this.tasks.size();
+    }
+
+    /**
+     * Returns a String which represents a formatted list of tasks in the bot.
+     *
+     * @return Formatted List of task in the bot's application.
+     */
+    public String printToList() {
```

Comment:
the return type and the method name seems to be a little off. I would expect a printToList() method to print the output in the console, instead of return a String